Formula/wsk.rb

```diff
@@ -0,0 +1,27 @@
+class Wsk < Formula
+  desc "OpenWhisk Command-Line Interface (CLI)"
+  homepage "http://openwhisk.org"
+  url "https://github.com/apache/incubator-openwhisk-cli/archive/0.9.0-incubating.tar.gz"
+  version "0.9.0-incubating"
+  sha256 "e2101daae8c4fdd43bb94bc9f8a2a29b74ce0a320b72e162f7f519b4dcbdd9b0"
+
+  depends_on "go" => :build
+  depends_on "go-bindata" => :build
+  depends_on "gradle" => :build
+
+  def install
+    system "sed", "-i.bak", "s/4\.1/4.9/", "./gradle/wrapper/gradle-wrapper.properties"
```

Comment:
What's going on with this, by the way?

Reply:
Gradle 4.1 triggers an error in the build, this bumps the Gradle version. I’m trying to get it fixed upstream, but in the meantime this was my fix.
https://github.com/apache/incubator-openwhisk-cli/pull/360

Reply:
If we jump over to using `godep` (if possible) does it become unnecessary?

Reply:
I'm assuming `godep` is what `gradle` is basically throwing itself out to to handle the vendoring, based off the contents of the `.gitignore` file in particular, but I could be wrong.

Reply:
If we could use `godep` then this step would become unnecessary indeed.

Reply:
I'll check out the `go` situation here later today. The other PR was relatively easy to look over so maybe we'll get lucky on this one as well.